tests/lax_scipy_sparse_test.py

```diff
@@ -12,28 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from functools import partial
 import unittest
+from functools import partial
```

Comment:
Any reason for these reording of imports? Some kind of local linter you're running? All else being equal, I'd prefer leaving these as-is to avoid churn and make the true changes easier to see.

Reply:
Ya, was habitual fingers going to hotkeys to format file. Undid it.